docs/data-types/sorted-sets.md

```diff
@@ -163,32 +153,22 @@ The main commands to operate with lexicographical ranges are `ZRANGEBYLEX`,
 `ZREVRANGEBYLEX`, `ZREMRANGEBYLEX` and `ZLEXCOUNT`.
 
 For example, let's add again our list of famous hackers, but this time
-use a score of zero for all the elements:
-
-    > zadd hackers 0 "Alan Kay" 0 "Sophie Wilson" 0 "Richard Stallman" 0
-      "Anita Borg" 0 "Yukihiro Matsumoto" 0 "Hedy Lamarr" 0 "Claude Shannon"
-      0 "Linus Torvalds" 0 "Alan Turing"
-
-Because of the sorted sets ordering rules, they are already sorted
-lexicographically:
-
-    > zrange hackers 0 -1
-    1) "Alan Kay"
-    2) "Alan Turing"
-    3) "Anita Borg"
-    4) "Claude Shannon"
-    5) "Hedy Lamarr"
-    6) "Linus Torvalds"
-    7) "Richard Stallman"
-    8) "Sophie Wilson"
-    9) "Yukihiro Matsumoto"
-
-Using `ZRANGEBYLEX` we can ask for lexicographical ranges:
-
-    > zrangebylex hackers [B [P
-    1) "Claude Shannon"
-    2) "Hedy Lamarr"
-    3) "Linus Torvalds"
+use a score of zero for all the elements. We'll see that because of the sorted sets ordering rules, they are already sorted lexicographically. Using `ZRANGEBYLEX` we can ask for lexicographical ranges:
```

Comment:
```suggestion
using a score of zero for all the elements. We'll see that because of the sorted set's ordering rules, they are already sorted lexicographically. Using `ZRANGEBYLEX` we can ask for lexicographical ranges:
```